### PR TITLE
Update m365-campaigns-conditional-access.md

### PR DESCRIPTION
**Pull request by msbemba**
Preview policies are no longer available under conditional access and the customers are to create their own.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3213

**Comment by msbemba** [comment]
@Sirkkuw these options have changed on the conditional access page and the preview policies no longer exist. So I have a few changes, let me know if there is anything that needs to be modified here.

File: microsoft-365/campaigns/m365-campaigns-conditional-access.md [hunk 1]
```diff
@@ -66,16 +66,16 @@ To require that all users sign in with a second form of ID:
 
 ## Set up baseline policies
 
-1. Go to [Azure portal](https://portal.azure.com), and then navigate to **Azure Active Directory** \> **Conditional Access**.
-    
-    The baseline policies are listed on the page, and you can see that **Require MFA for admins** and **End user protection** are already enabled after you completed the steps in [require MFA](#require-mfa).
-
-    ![Page that lists baseline policies for conditional access.](../media/casettings.png)
-2. See the following specific instructions for each policy:
+1. Go to [Azure portal](https://portal.azure.com), and then navigate to **Azure Active Directory** \> **Conditional Access** to create a new **Policy**
```
Comment (JohanFreelancer9) [inline microsoft-365/campaigns/m365-campaigns-conditional-access.md]:
```suggestion
1. Go to the [Azure portal](https://portal.azure.com), and then navigate to **Azure Active Directory** \> **Conditional Access** to create a **new policy**.
```

File: microsoft-365/campaigns/m365-campaigns-conditional-access.md [hunk 2]
```diff
@@ -66,16 +66,16 @@ To require that all users sign in with a second form of ID:
...
 
+See the following specific instructions for each policy
```
Comment (JohanFreelancer9) [inline microsoft-365/campaigns/m365-campaigns-conditional-access.md]:
```suggestion
See the following specific instructions for each policy: <br>
```

File: microsoft-365/campaigns/m365-campaigns-conditional-access.md [hunk 3]
```diff
@@ -66,16 +66,16 @@ To require that all users sign in with a second form of ID:
...
     - [Require MFA for admins](https://docs.microsoft.com/azure/active-directory/conditional-access/howto-baseline-protect-administrators)
     - [Require MFA for users](https://docs.microsoft.com/azure/active-directory/conditional-access/howto-baseline-protect-end-users)  
     - [Block legacy authentication](https://docs.microsoft.com/azure/active-directory/conditional-access/howto-baseline-protect-legacy-auth)
     - [Require MFA for service management](https://docs.microsoft.com/azure/active-directory/conditional-access/howto-baseline-protect-azure)
+    
+> [!NOTE]
+> The Preview Policies no longer exists and the customer will need to create their own policy.
```
Comment (JohanFreelancer9) [inline microsoft-365/campaigns/m365-campaigns-conditional-access.md]:
```suggestion
> Preview policies no longer exist and users will need to create their own policies.
```

File: microsoft-365/campaigns/m365-campaigns-conditional-access.md [hunk 4]
```diff
@@ -66,16 +66,16 @@ To require that all users sign in with a second form of ID:
...
-1. Go to [Azure portal](https://portal.azure.com), and then navigate to **Azure Active Directory** \> **Conditional Access**.
-    
-    The baseline policies are listed on the page, and you can see that **Require MFA for admins** and **End user protection** are already enabled after you completed the steps in [require MFA](#require-mfa).
-
-    ![Page that lists baseline policies for conditional access.](../media/casettings.png)
-2. See the following specific instructions for each policy:
+1. Go to [Azure portal](https://portal.azure.com), and then navigate to **Azure Active Directory** \> **Conditional Access** to create a new **Policy**
 
+See the following specific instructions for each policy
     - [Require MFA for admins](https://docs.microsoft.com/azure/active-directory/conditional-access/howto-baseline-protect-administrators)
```
Comment (JohanFreelancer9) [inline microsoft-365/campaigns/m365-campaigns-conditional-access.md]:
```suggestion
    - [Require MFA for admins](https://docs.microsoft.com/azure/active-directory/conditional-access/howto-baseline-protect-administrators) <br>
```

File: microsoft-365/campaigns/m365-campaigns-conditional-access.md [hunk 5]
```diff
@@ -66,16 +66,16 @@ To require that all users sign in with a second form of ID:
...
     - [Require MFA for users](https://docs.microsoft.com/azure/active-directory/conditional-access/howto-baseline-protect-end-users)
```
Comment (JohanFreelancer9) [inline microsoft-365/campaigns/m365-campaigns-conditional-access.md]:
```suggestion
    - [Require MFA for users](https://docs.microsoft.com/azure/active-directory/conditional-access/howto-baseline-protect-end-users) <br>
```

File: microsoft-365/campaigns/m365-campaigns-conditional-access.md [hunk 6]
```diff
@@ -66,16 +66,16 @@ To require that all users sign in with a second form of ID:
...
-    The baseline policies are listed on the page, and you can see that **Require MFA for admins** and **End user protection** are already enabled after you completed the steps in [require MFA](#require-mfa).
-
-    ![Page that lists baseline policies for conditional access.](../media/casettings.png)
-2. See the following specific instructions for each policy:
+1. Go to [Azure portal](https://portal.azure.com), and then navigate to **Azure Active Directory** \> **Conditional Access** to create a new **Policy**
 
+See the following specific instructions for each policy
     - [Require MFA for admins](https://docs.microsoft.com/azure/active-directory/conditional-access/howto-baseline-protect-administrators)
     - [Require MFA for users](https://docs.microsoft.com/azure/active-directory/conditional-access/howto-baseline-protect-end-users)  
     - [Block legacy authentication](https://docs.microsoft.com/azure/active-directory/conditional-access/howto-baseline-protect-legacy-auth)
```
Comment (JohanFreelancer9) [inline microsoft-365/campaigns/m365-campaigns-conditional-access.md]:
```suggestion
    - [Block legacy authentication](https://docs.microsoft.com/azure/active-directory/conditional-access/howto-baseline-protect-legacy-auth) <br>
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Sirkkuw Please merge this PR. Thanks.

**Pull request merged**